foreman-selinux-enable

```diff
@@ -17,23 +19,24 @@ is_redhat_6() {
 for selinuxvariant in targeted
 do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
-    # Load policy
-    /usr/sbin/semanage module -S $selinuxvariant \
-      -a /usr/share/selinux/${selinuxvariant}/foreman.pp.bz2
+    /usr/sbin/semanage port -E > $TMP_PORTS
```

Comment:
I have just built on CentOS 6 (latest stable with updates) and I don't see any errors there. This was a clean install, therefore clean upgrade of the RPM package that I've built. No errors at all. `policycoreutils-python-2.0.83-24.el6.x86_64`